File: parser.rb
require 'csv'
csv_contents = CSV.read('Ethnocentrism Strategy Counts.csv')
#Ignore first bit
20.times { csv_contents.shift }
open('Netlogo.csv', 'w') do |f|
	f.puts "CC,CD,DD,DC"
	csv_contents.each do |row|
		f.print row[1],",",row[5],",",row[13],",",row[9],"\n"
	end
end

File: parser.py
import csv
with open('Ethnocentrism Strategy Counts.csv', 'rb') as csv_contents:
	reader = csv.reader(csv_contents, delimiter=',')

	#Ignore first bit
	for i in range(0,20):
		reader.next()
	
	f = open('Netlogo.csv', 'w')
	f.write("CC,CD,DD,DC\n")
	count = 0
	#Read crucial part of csv
	for row in reader:
		if count >= 1000:
			break
		count = count + 1
		f.write(row[1]+","+row[5]+","+row[13]+","+row[9]+"\n")